core/sys/windows/user32.odin

```diff
@@ -84,6 +85,16 @@ foreign user32 {
 	LoadCursorW :: proc(hInstance: HINSTANCE, lpCursorName: LPCWSTR) -> HCURSOR ---
 	LoadImageW :: proc(hInst: HINSTANCE, name: LPCWSTR, type: UINT, cx: c_int, cy: c_int, fuLoad: UINT) -> HANDLE ---
 
+	CreateIcon :: proc(hInstance: HINSTANCE, nWidth: c_int, nHeight: c_int, cPlanes: BYTE, cBitsPixel: BYTE, lpbANDbits: PBYTE, lpbXORbits: PBYTE) -> HICON ---
+	CreateIconFromResource :: proc(presbits: PBYTE, dwResSize: DWORD, fIcon: BOOL, dwVer: DWORD) -> HICON ---
+	DestroyIcon :: proc(hIcon: HICON) -> BOOL ---
+	DrawIcon :: proc(hDC: HDC, X: c_int, Y: c_int, hIcon: HICON) -> BOOL ---
+
+	CreateCursor :: proc(hInst: HINSTANCE, xHotSpot: c_int, yHotSpot: c_int, nWidth: c_int, nHeight: c_int, pvANDPlane: PVOID, pvXORPlane: PVOID) -> HCURSOR ---
+	DestroyCursor :: proc(hCursor: HCURSOR) -> BOOL ---
+	// ShowCursor somehow made the raylib fail with: D:/a/Odin/Odin/vendor/raylib/raylib.odin(1016:22) Redeclaration of foreign procedure 'ShowCursor' with different type signatures
+	// ShowCursor :: proc(bShow: BOOL) -> c_int ---
```

Comment:
Any suggestions on how to resolve this? Otherwise i can live without :p
https://github.com/Hyrtwol/Odin/actions/runs/8112971484/job/22175296361#step:9:11

Reply:
In the latest commit, I have done something that should fix it.

Reply:
Tx m8, rebased up :)